package.json

```diff
@@ -153,7 +153,8 @@
       "urlcat@3.1.0": "patches/urlcat@3.1.0.patch",
       "@chainsafe/as-sha256@0.3.1": "patches/@chainsafe__as-sha256@0.3.1.patch",
       "@protobufjs/inquire@1.1.0": "patches/@protobufjs__inquire@1.1.0.patch",
-      "react-devtools-inline@4.27.8": "patches/react-devtools-inline@4.27.8.patch"
+      "react-devtools-inline@4.27.8": "patches/react-devtools-inline@4.27.8.patch",
+      "@walletconnect/socket-transport@1.7.8": "patches/@walletconnect__socket-transport@1.7.8.patch"
```

Comment:
@guanbinrui please document this in patches/readme.md and open an issue in the upstream

Reply:
The package is not maintained anymore. But we are still using it cause we are supporting WalletConnect V1.